bindings/matrix-sdk-crypto-js/src/requests.rs

```diff
@@ -256,12 +261,33 @@ pub struct KeysBackupRequest {
     pub body: JsString,
 }
 
+/** Other Requests * */
+
+/// Request that will publish a cross signing identity.
+///
+/// This uploads the public cross signing key triplet.
+#[wasm_bindgen(getter_with_clone)]
+#[derive(Debug)]
+pub struct SigningKeysUploadRequest {
+    /// The request ID.
+    #[wasm_bindgen(readonly)]
+    pub id: Option<JsString>,
+
+    /// A JSON-encoded object of form:
+    ///
+    /// ```json
+    /// {"master_key", …, "self_signing_key": …, "user_signing_key": …}
```

Comment:
that's not correct json ... but with `...` that never will be ...
```suggestion
    /// {"master_key": …, "self_signing_key": …, "user_signing_key": …}
```

Reply:
;-)